content/docs/guides/contribute/code.en.md

```diff
@@ -41,6 +41,8 @@ This constraint does not propagate through APIs: You can use OSRD as a library,
 
 This chapter is about the process of integrating changes into the common code base. **If you need help at any stage, open an issue or message us.**
 
+_The review process is proposed on a [dedidacted page](../code-reviews/#life-of-a-pull-request)._
```

Comment:
nit (same for 🇫🇷 ):
* I would move that into the "pull request" section.
* And avoid repetition by letting the mtlynch article only on one page (in the dedicated page?)

Reply:
As we said, I'll merge the pages, see ya soon for a new review ;)

Reply:
All has changed...!